experiments/vision-transformer/unetr/livecell/common.py

```diff
@@ -40,14 +53,100 @@ def _get_output_channels(with_affinities):
     return n_out
 
 
+def get_distance_maps(labels):
+    labels = vigra.analysis.relabelConsecutive(labels.astype("uint32"))[0]
+    # compute the eccentricty centers
+    # First expensive step: center computation (leave it here, it's done once per image)
+    centers = vigra.filters.eccentricityCenters(labels.astype("uint32"))
+
+    # 1 for all eccentricity centers of the cells 0 elsewhere
+    center_mask = np.zeros_like(labels)
+    for center in centers:
+        center_mask[center] = 1
+
+    x_distances = np.zeros(labels.shape, dtype="float32")
+    y_distances = np.zeros(labels.shape, dtype="float32")
+
+    _, bbox_coordinates = get_centers_and_bounding_boxes(labels, mode="p")
+
+    def compute_distance_map(cell_id):
+        mask = labels == cell_id
+
+        # getting the bounding box coordinates for masking the roi
+        bbox = bbox_coordinates[cell_id]
+
+        # crop the respective inputs to the bbox shape (for getting the distance transforms in the roi)
+        cropped_center_mask = center_mask[
```

Comment:
This is wrong! It gets all the centers in the bounding box, so it will not compute the specific distances for this cell.